straxen/contexts.py

```diff
@@ -47,6 +47,8 @@
     s1_max_rise_time=100,
     s2_xy_correction_map=("CMT_model", ('s2_xy_map', "ONLINE"), True),
     elife_file=("elife_model", "ONLINE",True),
+    fdc_map=("CMT_model", ('fdc_map', "ONLINE"), True),
+    electron_drift_velocity=0.632e-4,
```

Comment:
@ahiguera-mx , Would you agree that it's better to track in CMT?

Reply:
yes, see my comment above

Reply:
@ftoschi since you have one map for every single reco alg, I guess the option should be something like 
`fdc_map=("CMT_model", ('fdc_map', "ONLINE"), True)`